src/cranbtree.c

```diff
@@ -256,8 +259,11 @@ void *cbt_search(cranbtree_t * bt, int key)
   */
 void *cbt_delete(cranbtree_t * bt, int key)
 {
-	assert(bt != NULL);
-
+	//assert(bt != NULL);
+	if (bt == NULL)
```

Comment:
This error should be set when cbt_delete have tried to delete an object with the given key but did not find it. However, if bt is NULL it means that the user of the library has a bug in his code and he is passing NULL to cbt_delete.